### DISCUSSION TITLE
Quitting SwiftBars own menu item spelled wrong

### DISCUSSION THREAD

**Discussion by bicaluv** [swiftbar/SwiftBar#299] [General]
Hi!
In version 1.4.3, the menu item for quitting SwiftBar is named "Quite". shouldn't it be named "Quit"?
<img width="202" alt="image" src="https://user-images.githubusercontent.com/17777937/159758914-7a490c52-6751-4725-b91a-9855556a5cbc.png">

**Comment by melonamin** [comment]
Thnx, fixed
